### DISCUSSION TITLE
[Question] How to access Schemathesis container running on remote server?

### DISCUSSION THREAD

**Discussion by illuminato8** [schemathesis/schemathesis#1827] [Q&A]
I have a question about how I would access Schemathesis if it ran on a remote server.

On remote server I have docker-compose file:
```
version: '3'
services:
  schemathesis:
    tty: true
    build: ./schemathesis
    ports:
      - "8888:80"
```

And also I have folder with Schemathesis repo where docker container located. I removed last line (`" ENTRYPOINT ["schemathesis"]"`) from docker container so I do not need rebuild container each time. 

I run schemathesis as following:
```
docker-compose up -d
docker-compose exec schemathesis sh
schemathesis run --checks all https://example.schemathesis.io/openapi.json
```

It works perfectly. Now I want to access this remote server from PC. Let assume server's IP is `10.10.100.11`. 

I am looking for a way to make POST requests to this server using e.g. the following script:
```
url = 'http://110.10.100.11:8888'
command: 'schemathesis run --checks all https://example.schemathesis.io/openapi.json'
response = requests.post(url=url, data=command)
```

But I am not sure what I need to add to the remote server to run the script successfully. Any help would be appreciated. Thanks.

**Comment by Stranger6667** [comment]
Sorry for the late and not helpful reply, but unfortunately I don't know how to resolve this issue. It seems to be Docker-specific and I am not that proficient with it.
